packages/ra-ui-materialui/src/input/AutocompleteArrayInput.js

```diff
@@ -247,6 +250,11 @@ export class AutocompleteArrayInput extends React.Component {
             ref(input);
         };
 
+        const finalOptions = {
+            fullWidth,
+            ...options,
```

Comment:
So we ignore all others options?

Reply:
Have you missed the `...options` ?

Reply:
Nvm I thought it was destructuring, but it isn't !